### PR TITLE
Integer handling for feasible sets

### PR DESCRIPTION
**Pull request by Ovascos**
This PR adds support for query the number of integers in a feasible set.

### CODE REVIEW

File: src/polynomial/feasibility_set.c [hunk 1]
```diff
@@ -144,11 +144,37 @@ int lp_feasibility_set_is_point(const lp_feasibility_set_t* set) {
   return set->size == 1 && set->intervals->is_point;
 }
 
+int lp_feasibility_set_is_point_int(const lp_feasibility_set_t* set) {
+  long cnt = 0;
+  for (size_t i = 0; i < set->size; ++i) {
+    long tmp = lp_interval_count_int(set->intervals + i);
```
Comment (ahmed-irfan) [inline src/polynomial/feasibility_set.c:150]: wondering if we can check this without counting?

File: src/polynomial/feasibility_set.c [hunk 2]
```diff
@@ -144,11 +144,37 @@ int lp_feasibility_set_is_point(const lp_feasibility_set_t* set) {
...
+    assert(tmp >= 0);
+    if (tmp > 1 || tmp + cnt > 1) {
```
Comment (ahmed-irfan) [inline src/polynomial/feasibility_set.c:152]: I think this is handling the overflow case. Right?
Can you add a comment here? thanks

File: src/polynomial/feasibility_set.c [hunk 3]
```diff
@@ -170,9 +196,27 @@ char* lp_feasibility_set_to_string(const lp_feasibility_set_t* set) {
 }
 
 int lp_feasibility_set_contains(const lp_feasibility_set_t* set, const lp_value_t* value) {
-  // TODO: binary search
```
Comment (ahmed-irfan) [inline src/polynomial/feasibility_set.c:173]: nice

File: src/polynomial/feasibility_set.c (hunk 1 above)
Comment (Ovascos) [reply src/polynomial/feasibility_set.c:150]: Counting is more or less for free, as it is "just" one subtraction per interval.

**Review by ahmed-irfan — APPROVED** [review]
LGTM


Thanks

**Pull request merged**